app/design/frontend/base/default/template/boltpay/replace.phtml

```diff
@@ -42,23 +44,28 @@ if(!Mage::helper('boltpay')->canUseBolt($this->getQuote(), false)) return;
          ****************************************************************************************************/
         var replaceCheckout = function() {
 
-            var replaceRetries  = 20;
-            var replaceInterval = 500;
-
+            let replaceRetries  = 20,
+                replaceInterval = 500;
 
             // replaceable button selectors and types of Bolt buttons to replace them with
-            var selectors = {
+            let selectors = {
```

Comment:
I tend to be conservative introducing new ES syntax (well let is not "new" but there are a few legacy browsers that don't support it). I suggest we stick to 'var'.

Reply:
ok